be/src/storage/vectorized/delta_writer.cpp

```diff
@@ -281,7 +281,6 @@ Status DeltaWriter::close_wait(google::protobuf::RepeatedPtrField<PTabletInfo>*
     _delta_written_success = true;
 
     const FlushStatistic& stat = _flush_token->get_stats();
-    LOG(INFO) << "Closed delta writer. tablet_id=" << _tablet->tablet_id() << " stats=" << stat;
```

Comment:
Flush time, count, size are usefull, better not to remove.

Reply:
> Flush time, count, size are usefull, better not to remove.

updated, PTAL